### PR TITLE
- added function: Dose From Fluence, Dose From Fluence Single

### PR DESCRIPTION
**Pull request by marwin1991**
Fluence for plotting should be the same for every energy or also generated or Energy should be constant ?

### CODE REVIEW

File: src/static/json/PhysicsRoutines/DoseFromFluence.json [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+      "asManyAsPoints": true,
+      "defaultValue": 1001,
+      "description": "Choose particle type"
+    },
+    {
+      "type": "input",
+      "parameterName": "fluence_cm2",
+      "label": "Fluence",
+      "placeholder": "1",
+      "defaultValue": 1,
```
Comment (grzanka) [inline src/static/json/PhysicsRoutines/DoseFromFluence.json]: default value should be 10^8

File: src/static/json/PhysicsRoutines/DoseFromFluenceSingle.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  "description": "Returns dose in Gy for particle with given fluence and energy",
+  "plot": false,
+  "unit": "Gy",
+  "formItems": [
+    {
+      "type": "input",
+      "parameterName": "E_MeV_u",
+      "label": "Energy",
+      "placeholder": "10",
+      "defaultValue": 10,
```
Comment (grzanka) [inline src/static/json/PhysicsRoutines/DoseFromFluenceSingle.json]: Default energy should be 150

File: src/static/json/PhysicsRoutines/DoseFromFluenceSingle.json [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+      "asManyAsPoints": false,
+      "defaultValue": 1001,
+      "description": "Choose particle type"
+    },
+    {
+      "type": "input",
+      "parameterName": "fluence_cm2",
+      "label": "Fluence",
+      "placeholder": "1",
+      "defaultValue": 1,
```
Comment (grzanka) [inline src/static/json/PhysicsRoutines/DoseFromFluenceSingle.json]: Default 10^8, typical fluence range is 10^4 ... 10^12

**Comment by marwin1991** [comment]
Fixes #57

**Comment by adudzik** [comment]
It looks like after merging application was built but not deployed. See #63.

**Pull request merged**